security/cert_manager_operator/cert-manager-operator-release-notes.adoc

```diff
@@ -12,6 +12,34 @@ These release notes track the development of {cert-manager-operator}.
 
 For more information, see xref:../../security/cert_manager_operator/index.adoc#cert-manager-operator-about[About the {cert-manager-operator}].
 
+[id="cert-manager-operator-release-notes-1-14-0"]
+== {cert-manager-operator} 1.14.0
+
+Issued: 2024-07-08
+
+The following advisory is available for the {cert-manager-operator} 1.14:
+
+* link:https://errata.engineering.redhat.com/advisory/128547[RHEA-2024:128547-02]
```

Comment:
This should be an access.redhat.com IIRC for ensuring it's publicly accessible to customers once Errata is shipped.

Reply:
I think a LIVE ID needs to be assigned to the advisory:

![Screenshot from 2024-07-05 12-34-35](https://github.com/openshift/openshift-docs/assets/57954076/652c4afb-0739-4f9b-b66f-a5c112a361ec)


This would be common practice for core OCP RNs. [Example](https://docs.openshift.com/container-platform/4.16/release_notes/ocp-4-16-release-notes.html#ocp-4-16-0-ga_release-notes). Notice the shorter advisory names.

Reply:
The link you created might not render after the advisory is shipped.I think this is why the LIVE ID is needed.

Reply:
I have updated the access.redhat link which renders properly.